examples/supplychain-app/README.md

```diff
@@ -26,8 +26,8 @@ Build the following docker images from the respective folders and store in the d
 * supplychain_frontend:latest (from `examples/supplychain-app/supplychain-frontend/` folder)
 * supplychain_corda:springboot_latest (from `examples/supplychain-app/corda/cordApps_springBoot/` folder) for Corda
 * supplychain_corda:express_app_latest (from `examples/supplychain-app/corda/express_nodeJS/` folder) for Corda
-* supplychain_fabric:rest_server_latest (from `examples/supplychain-app/fabric/chaincode_rest_server/rest-server/` folder) for Fabric
-* supplychain_fabric:express_app_latest (from `examples/supplychain-app/fabric/express_nodeJs/` folder) for Fabric
+* supplychain_fabric/rest_server:latest (from `examples/supplychain-app/fabric/chaincode_rest_server/rest-server/` folder) for Fabric
```

Comment:
these are not correct images from dockerhub

Reply:
Did not realise these were from the official Hyperledger Labs Docker Hub (you mean [this link](https://hub.docker.com/r/hyperledgerlabs/supplychain_fabric/tags) right?). I'll change it back

Reply:
Changed in [c44e8bf](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1096/commits/c44e8bfc500eb45912b5fb9988085bb0eb99e50c)

Reply:
cant see the changes

Reply:
Probably mistake in `git reset $(git merge-base)` command. Re-added in commit [95916d1](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1096/commits/95916d1d21a7b0cc4ed0c8ac7087c1db6ad2ce0f)